src/components/Login/LoginOTPCard.tsx

```diff
@@ -69,11 +75,42 @@ export const LoginOTPCard: FunctionComponent<LoginOTPCard> = ({
     try {
       const response = await validateOTP(otp, mobileNumber, codeKey, endpoint);
       setIsLoading(false);
-      setAuthInfo(response.sessionToken, response.ttl.getTime(), endpoint);
-      navigation.navigate("CollectCustomerDetailsScreen");
+      const versionResponse = await getEnvVersion(
+        response.sessionToken,
+        endpoint
+      );
+
+      // Toggle between different environments
+      // using the DIST_ENV variable from features
+
+      switch (versionResponse.features.DIST_ENV) {
+        case "DEFAULT":
+          setAuthInfo(response.sessionToken, response.ttl.getTime(), endpoint);
+          setFeatures(versionResponse.features);
+          setProducts(versionResponse.policies);
+          navigation.navigate("CollectCustomerDetailsScreen");
+          break;
+
+        // TODO: Integration with merchant flow
```

Comment:
Not done in this PR?

Reply:
Regardless, can leave `case "MERCHANT"` uncommented actually, stacked with `default:`

Reply:
okay~ im leaving it here for sebs + interns to integrate, thankss!

Reply:
Haha I meant the case "MERCHANT" can be uncommented actually

Reply:
haha yeah i know! i mean im leaving it so they will do it during integration, ps for being unclear and thanks!!!🙏